modulefiles/ufs_hercules.gnu.lua

```diff
@@ -11,8 +11,8 @@ load(pathJoin("stack-gcc", stack_gnu_ver))
 stack_openmpi_ver=os.getenv("stack_openmpi_ver") or "4.1.5"
 load(pathJoin("stack-openmpi", stack_openmpi_ver))
 
-stack_python_ver=os.getenv("stack_python_ver") or "3.9.14"
-load(pathJoin("stack-python", stack_python_ver))
+--stack_python_ver=os.getenv("stack_python_ver") or "3.10.8"
+--load(pathJoin("stack-python", stack_python_ver))
```

Comment:
Lines 20-47 could be removed, and replaced by `load("ufs_common")`

Reply:
Thanks @natalie-perlin I just made this change for Hercules with Intel and GNU.